package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "landos",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "prettier": "prettier"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/WapBly/Landos.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/WapBly/Landos/issues"
+  },
+  "homepage": "https://github.com/WapBly/Landos#readme",
+  "dependencies": {
```

Comment:
Please rename `dependencies` to `devDependencies`, because packages `prettier` and `stylelint` used only when you create site but not when you publish site.

After rename run in console `npm install`